### ISSUE TITLE
We need design books

### ISSUE THREAD

**Issue by EOEboh** [EOEboh/Free-Software-Books#39]
This repo isn't just aimed at developer-centric folks, we need the resources to be as inclusive as possible. We will really appreciate contributions on design, UX, UI and product design books.

**Comment by binadiegha** [comment]
I believe I have one good UX book. would set the pace

**Comment by binadiegha** [comment]
created the design section and added a UX book ✅

**Comment by EOEboh** [comment]
@binadiegha how far? Do you have more books on design? We need to discuss how to onboard more design books to this project

**Comment by binadiegha** [comment]
> @binadiegha how far? Do you have more books on design? We need to discuss how to onboard more design books to this project

Not really. I lost most of my design resources in my old pc. But I'll take up the task to source for them 🙏🏾🙌🏾
